__tests__/app.test.js

```diff
@@ -168,3 +168,74 @@ describe("GET /api/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("Expect 201 when given valid article id and response body should return the posted comment", () => {
```

Comment:
could add another 201 test to check that any unnecessary properties on the request body are ignored i.e.

`.send({ username: "phil", body: "yes", ignoreThis: "hmmm" })`

would still result in a successful comment post